.github/workflows/e2e.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
 
     services:
       mongo:
-        image: mongo:3.4.23
+        image: mongo:4.4.3
```

Comment:
Do we want to just set this to `mongo` without a version like we do in the docker-compose file?

Reply:
Probably, actually. Good catch.